plugins/pulp_rpm/plugins/importers/yum/sync.py

```diff
@@ -615,6 +617,8 @@ def add_rpm_unit(self, metadata_files, unit):
             unit.save()
         except NotUniqueError:
             unit = unit.__class__.objects.filter(**unit.unit_key).first()
+        if rpm_parse.signature_enabled(self.config):
```

Comment:
@ipanova +1 to your general idea about log statements.

I think each failure to import due to signature check should log at DEBUG level, which will cause almost all users to never see huge amounts of log statements which is good. The single line statement you've written look good and that I think should be at WARNING or INFO. I think I'm leaning to WARNING.

I think the progress report needs to report each unit that was not associated due to signature check failure. My questions are: how do we report failed unit imports today? Do we even? I know there is an `error_details` section of the progress reports and it probably is good if each failure is named in there.

Also there is a general error message in the progress reporting which would be great to contain a similar message as the '12345 RPMS failed the signature ....'. The general error message is important because pulp-admin shows only the general one not the long specific ones.

OK back to the log statement. I think if the progress report contains all of the right data then there should be an easy opportunity later during the sync/import/upload which looks for these types of errors in the error_details, counts them, and logs the total later. That would be after all of the units have been processed in the loop so by then you'll have the full knowledge you need.